src/plugins/index_pattern_management/public/components/edit_index_pattern/indexed_fields_table/__snapshots__/indexed_fields_table.test.tsx.snap

```diff
@@ -125,7 +125,7 @@ exports[`IndexedFieldsTable IndexedFieldsTable with rollup index pattern should
           "hasRuntime": false,
           "info": Array [
             "Rollup aggregations:",
-            "histogram (interval: 5)",
+            "histogram (interval: )",
```

Comment:
not sure if this change is expected and if the test now testing what it says it testing. could you double-check this one? if the test is redundant now, I'd remove

Reply:
Good catch! I need to fix this. Likely the result of a bad merge

Reply:
addressed